style.css

```diff
@@ -607,3 +611,308 @@ ul {
   position: absolute;
   left: 0;
 }
+
+/* Desktop-version */
+
+@media (min-width: 768px) {
+  body {
+    margin: 0;
+  }
+
+  footer {
+    margin: 0;
+  }
+
+  .button1 {
+    display: none;
+  }
+
+  .first-img {
+    display: none;
+  }
+
+  .orders {
+    display: flex;
+    gap: 15px;
+    color: #344563;
+    text-decoration: none;
+    margin-top: 2%;
+    margin-right: 7%;
+  }
+
+  .container {
+    position: relative;
+    width: 100%;
+  }
+
+  .linkers {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    margin-right: 3%;
+  }
+
+  .section-2 {
+    background: url(Microverse-Student-Project-1/header.png) no-repeat;
+    background-size: 99%;
+    background-position-x: right;
+    background-color: #fff;
+    display: flex;
+    flex-direction: column;
+    height: 1000px;
+    width: 100%;
+    border-bottom-left-radius: 70px;
+    margin-bottom: 0;
+  }
+
+  .paragraph {
+    margin-top: 100px;
+    margin-left: 20%;
+    margin-right: 390px;
+    text-align: left;
+    width: 50%;
+  }
+
+  .section-3 {
+    display: grid;
+    vertical-align: bottom;
+    margin-top: 30px;
+    margin-bottom: 30px;
+
+    /* margin-left: 10px; */
+    margin-right: 10px;
+
+    /* gap: 1%; */
+
+    /* padding: 142px; */
+    width: 100%;
+    height: 2694px;
+    position: relative;
+
+    /* left: 30px; */
+  }
+
+  .card {
+    width: 70%;
+    height: 496px;
+    display: flex;
+    flex-direction: row;
+    border-radius: 24px;
+    gap: 50px;
+    position: relative;
+    left: 12%;
+    margin: 10% 15% 0 3%;
+  }
+
+  .photo {
+    width: 45%;
+    border-radius: 8px;
+    margin-right: 3px;
+  }
+
+  .work-img {
+    width: 100%;
+    height: 448px;
+  }
+
+  .left-block {
+    position: absolute;
+    right: 5%;
+    top: 30px;
+    gap: 8px;
+    width: 45%;
+    margin-left: 3px;
+  }
+
+  .work-img1 {
+    width: 45%;
+    height: 448px;
+    border-radius: 8px;
+    position: absolute;
+    right: 2%;
+    top: 30px;
+  }
+
+  .left-block1 {
+    position: absolute;
+    left: 2%;
+    top: 30px;
+  }
+
+  .des-text {
+    width: 100%;
+    height: 30%;
+    margin-top: 40px;
+  }
+
+  /* about my self section */
+
+  .desk-continer {
+    width: 100%;
+    height: 942px;
+    background-color: #fff;
+    border-top-right-radius: 80px;
+    border-top-right-radius: 10%;
+  }
+
+  .about-my-self {
+    display: none;
+    width: 1500px;
+    height: 942px;
+    position: absolute;
+    z-index: -1;
+  }
+
+  .skill {
+    position: absolute;
+    width: 50%;
+
+    /* right: 90px; */
+    left: 50%;
+    top: 530%;
+  }
```

Comment:
- [ ] Your "About Myself" and "Skills" sections are overlapping the next section on some screen sizes. Kindly try to position elements on the page using margins and padding following the dimensions that the Figma template provides, and be cautious when using top/left/bottom/right especially when using "percentage dimensions".
![image](https://user-images.githubusercontent.com/61919617/171957838-585f3129-25ba-405c-8739-a49ec79b0db3.png)

Reply:
hello dear 
everything is working fine with me 
can you tell me what is the size that you used to check the project???
i used 768 and wider by using google can you please tell me , 
or maybe check again
@leonardodiasb

Reply:
Hello,
This was around 1000px wide. This was caused because you used `top: 530%;` on both "top" and "skill" classes. Try removing this top attribute and use margin to position these elements.

Reply:
hi 
i will do it but
this is the only problem right? 
because im going to use extra review request  |
thanks @leonardodiasb

Reply:
The problem is that some elements of your portfolio are overlapping on some screen sizes. By doing this you may fix this issue. But I advise you to check by yourself the responsiveness of your app by changing the screen width. You can open the developer tools of your browser, open the Responsive Design Mode and change the width, or just unmaximize the window and stretch it. 👍